### ISSUE TITLE
Is this still supported/working

### ISSUE THREAD

**Issue by cwhittl** [jgclark/NotePlan-tools#42]
I see that most of the functionallity was moved to the plugin but I'm really only interested in the repeats scripting.  Due to some of the limitations of triggers not firing when completing tasks in a filter or daily note I need something to run on my machine to basically redo those repeats.  So is that part of the ruby script still working?

**Comment by jgclark** [comment]
Yes, all the functionality here should be available now in the various plugins. This script is not maintained, though I can't think of any reason why this script's repeat functionality shouldn't still work fine.

I too sometimes have repeats that are completed when the editor isn't open, so triggers don't fire. To solve that I recently added a new "/Generate @repeats..." command to the Tidy plugin that generates any needed repeats from all notes changed in the last N days.
I would hope that covers all your cases, if you run that each day.
Or, as I do, include it my daily template.

**Comment by cwhittl** [comment]
Thanks, I have used that but the only issue is that every doc that it hits that doesn't it pops up and requires you to hit ok before it continues.  Not sure if this is an issue with the plugin or changes in the app.

**Comment by jgclark** [comment]
Sorry I don't follow your first sentence: "that it doesn't" what?. And what pops up?

**Comment by cwhittl** [comment]
When i run the Tidy/regen code, I get this
![image](https://github.com/jgclark/NotePlan-tools/assets/764314/aa40e0c8-9e37-4192-9a6a-70e6bb67d591)
 and this ![image](https://github.com/jgclark/NotePlan-tools/assets/764314/01598f5d-f9be-4111-91b0-34161de0f058)
for each file it goes against.

I have to click OK before it goes to the next one. 

Is there a way to disable the prompt?

**Comment by jgclark** [comment]
Yes. The documentation shows there's a run silently option when run manually or through a template.

**Comment by cwhittl** [comment]
Sorry to bother again, the only silent option I can find is in Tidy and it's set to true but still throws the error (I'm in IPADOS)

![image](https://github.com/jgclark/NotePlan-tools/assets/764314/ba56fc8d-e913-4f74-b2d5-93c21f6b1242)


![image](https://github.com/jgclark/NotePlan-tools/assets/764314/881fa1f7-5e31-477a-9fc4-fd523d680f7b)

**Comment by jgclark** [comment]
OK, I can't tell where the problem is, and it's much harder to debug something on iPad, as its hard to get see the DEBUG level logging. 
What happens when you try the specific "/Generate @repeats in recent notes" command?

**Comment by cwhittl** [comment]
I tried it on my Ipad and Mac and both keep prompting.  I am on the beta of Notepad so not sure that matters or not but I think the issue seems to be that it's not reading the config correctly.  I

**Comment by jgclark** [comment]
Just checking in to see if this is still an issue, @cwhittl? There have been a number of releases in the last year, which may have resolved this.
